### DISCUSSION TITLE
Hello everyone! I need some help. I have a problem when I set up cvxpy v.1.4.1 on python 3.12

### DISCUSSION THREAD

**Discussion by HaMinhTuan1986** [cvxpy/cvxpy#2266] [Q&A]
I have a problem when I set up cvxpy v.1.4.1 on python 3.12. This is message error. Can you help me, please!
Collecting scs>=3.0 (from cvxpy)
  Using cached scs-3.2.3.tar.gz (1.2 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error

**Comment by RoyiAvital** [comment]
Are you on Windows? 

It may be related to https://github.com/cvxpy/cvxpy/issues/2269.

**Comment by phschiele** [comment]
I'm closing this discussion as CVXPY + all dependencies have wheels for Python 3.12 (e.g., CVXPY 1.4.3).
